doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -192,7 +192,50 @@ Bluetooth samples
 Bluetooth mesh samples
 ----------------------
 
-|no_changes_yet_note|
+Bluetooth mesh samples
+----------------------
+
+* :ref:`bluetooth_ble_peripheral_lbs_coex` sample:
+
+  * Updated:
+
+    * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option.
+
+* :ref:`bt_mesh_chat` sample:
+
+  * Updated:
+
+    * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option.
+
+* :ref:`bluetooth_mesh_light` sample:
+
+  * Updated:
+
+    * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option.
+
+* :ref:`bluetooth_mesh_light_switch` sample:
+
+  * Updated:
+
+    * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option.
+
+* :ref:`bluetooth_mesh_sensor_client` sample:
+
+  * Updated:
+
+    * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option.
+
+* :ref:`bluetooth_mesh_sensor_server` sample:
+
+  * Updated:
+
+    * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option.
+
+* :ref:`bluetooth_mesh_light_lc` sample:
+
+  * Updated:
+
+    * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option.
```

Comment:
Do we need to do it this way with one line per sample, or is it possible to add one line like "enabled PARTIAL_ERASE to all relevant sample" to cover all in a more readable way?

Reply:
There are no clear rules for this, but it can of course be done e.g. like this:

* Updated:

  * Enabled :kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` configuration option for the following Bluetooth mesh samples:

    * :ref:`bluetooth_ble_peripheral_lbs_coex`
    * xx
    * xx
    * ...

If we are planning to do other changes to a few of those samples, it might be ok to leave it the way it is. But if we don't have other changes planned, we might as well consider the other option, maybe it looks neater.

Reply:
I will leave this as is. Based on the backlog we will likely have some more sample updates.